ui/pages/onboarding-flow/privacy-settings/privacy-settings.js

```diff
@@ -321,7 +319,11 @@ export default function PrivacySettings() {
             ])}
           />
         </div>
-        <Button type="primary" rounded onClick={handleSubmit}>
+        <Button
+          variant={BUTTON_VARIANT.PRIMARY}
```

Comment:
Maybe add size here too then it should match
```suggestion
          variant={BUTTON_VARIANT.PRIMARY}
          size={BUTTON_SIZES.LG}
```